app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -135,7 +136,16 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
                     onNavigationEvent?.invoke(NavigationEvent.LOAD_TILE, urlStr, null)
                 }
             }
-        }, onTileLongClick = openHomeTileContextMenu)
+        }, onTileLongClick = openHomeTileContextMenu, onTileFocused = {
+            val prefInt = PreferenceManager.getDefaultSharedPreferences(context).getInt(SHOW_UNPIN_TOAST_COUNTER_PREF, 0)
```

Comment:
Hm, I guess this shows the toast 3 times on a single launch. I don't think that really makes sense - probably my fault for not being entirely clear "when tile is first selected on a launch, max 3 times".

The approach to use sharedprefs seems fine, but let me clarify this behavior in the bug.

Reply:
Can you add a constant that gets set once per launch, maybe in `initTiles`, or in one of the `onCreate*`?